### DISCUSSION TITLE
如何下载一篇国标？

### DISCUSSION THREAD

**Discussion by giscus-xzonn** [Xzonn/xzonn.github.io#13] [General]
# 如何下载一篇国标？

如何下载一篇国标？ - 来自Xzonn的小站

https://xzonn.top/posts/How-to-Download-Chinese-Standard.html

**Comment by Xzonn** [comment]
> 评论导出自来必力

*****塩鱼 评论于 2020-06-06 01:05:09 (UTC+8)*****

太厉害了！！！

**Comment by Xzonn** [comment]
> 评论导出自来必力

*****tutu 评论于 2020-08-26 15:20:06 (UTC+8)*****

参照以上方法，百度文库pdf有没有可能下载？

**Comment by Xzonn** [comment]
> 评论导出自来必力

*****3qsami 评论于 2021-02-07 19:36:22 (UTC+8)*****

PDFViewerApplication.download();  
报错了：Uncaught TypeError: PDFViewerApplication.download is not a function

*****塩魚 回复于 2021-03-19 11:17:54 (UTC+8)*****

报错了+1，大佬什么时候再解决一下

**Comment by Xzonn** [comment]
> 评论导出自来必力

*****erdf 评论于 2021-04-12 23:20:34 (UTC+8)*****

PDFViewerApplication.download();好像失效了  
能否求大佬更新下方法，谢谢

**Comment by Xzonn** [comment]
> 评论导出自来必力

*****Xzonn 评论于 2021-05-26 14:18:12 (UTC+8)*****

已经更新了方法，不保证长期有效。

**Comment by Xzonn** [comment]
> 评论导出自来必力

*****刘能胜 评论于 2021-10-25 19:40:36 (UTC+8)*****

大佬太厉害了

**Comment by Xzonn** [comment]
> 评论导出自来必力

*****空无 评论于 2021-11-24 13:12:04 (UTC+8)*****

牛逼

**Comment by Xzonn** [comment]
> 评论导出自来必力

*****pj 评论于 2021-11-30 11:44:19 (UTC+8)*****

您好，https://www.spc.org.cn/这个网站是用福昕阅读器在线加密打开的，用这个代码不能下载，有没有办法呢？

**Comment by Xzonn** [comment]
> 评论导出自来必力

*****eirr 评论于 2022-04-26 09:28:02 (UTC+8)*****

大佬你好，现在国标下载好像又改版了，之前的代码又失效了，能麻烦再看看吗

**Comment by Xzonn** [comment]
> 评论导出自来必力

*****戬邪 评论于 2023-03-27 12:00:22 (UTC+8)*****

最新的国标预览已经改成了PNG错位加载，每一个请求加载的都是一个png图片有点向游戏的贴图加载。
